### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by rogeralsing**
Remove netstandard 2.1
Restructure solution structure
Remove Proto.Remote.GrpcNet and Proto.Repote.GrpcCore in favor of just Proto.Remote containing all grpcnet code

### CODE REVIEW

**Comment by rogeralsing** [comment]
Also, any idea why some tests consistently fail? or maybe just CI acting up

**Comment by mhelleborg** [comment]
> Also, any idea why some tests consistently fail? or maybe just CI acting up

I did rewrite the tests to run concurrently. Might need higher timeouts because of that, since they might get CPU throttled

File: src/Proto.Remote/GrpcNet/HostedGrpcNetRemote.cs [hunk 1]
```diff
@@ -85,4 +91,62 @@ public Task ShutdownAsync(bool graceful = true)
...
\ No newline at end of file
+}
+
+
+
+
+namespace Proto.Remote.GrpcNet
+{
+    [PublicAPI]
+    public static class Extensions2
```
Comment (mhelleborg) [inline src/Proto.Remote/GrpcNet/HostedGrpcNetRemote.cs]: ServiceCollectionExtensions?

**Comment by rogeralsing** [comment]
I've disabled everything else but the two failing tests. still fails.
Very unclear why. one just seem to get stuck, not doing anything?

**Pull request merged**